misaligned.c

```diff
@@ -15,7 +14,10 @@ int printColorMap() {
 
 int main() {
     int result = printColorMap();
+    assert(result==-25);
+    assert(result==0);

```

Comment:
The asserts are meant to be statements of functionality. These asserts will fail even when we fix the code